### ISSUE TITLE
[req] - "sv_alltalk 6"

### ISSUE THREAD

**Issue by fred0r** [rehlds/ReGameDLL_CS#645]
"sv_alltalk 6"

![file](https://user-images.githubusercontent.com/2220111/122484560-258e5280-cfd5-11eb-9981-884644ff750d.png)

Also a "sv_alltalk 7" would be very nice

![alltalk_7](https://user-images.githubusercontent.com/2220111/122484984-2ecbef00-cfd6-11eb-8381-a5f1b6796453.png)

These Modes would be very usefull on large Servers.

**Comment by fred0r** [comment]
@MatiasEsf
On Server with 20+ Players is quite hard to follow the Voice-Chat when each Team is also able to hear the alive Opponents of both Sides.

**Comment by wilianmaique** [comment]
or add: sv_alltalk_enable_custom 1

and add variables of each type

sv_alltalk_custom_ct_hear_alive "1|0"
sv_alltalk_custom_ct_hear_dead "1|0" 
etc..

so each person assembles their model as they wish.

**Comment by fred0r** [comment]
or maybe being able to change to volume on dead/alive players

**Comment by Nord1cWarr1or** [comment]
Duplicate of #387/#228 — the requested `sv_alltalk 6/7` voice matrices are exactly what's tracked as the pending `case 6` addition to `CanPlayerHearPlayer` (`multiplay_gamerules.cpp:102`); this issue independently requests the same modes. Closing as not planned — see #387/#228 for the feature queue.
